_pages/order-lib.md

```diff
@@ -4,19 +4,46 @@ title: Order Lib Release Notes
 
 Spartacus 4.2 introduces the Order Library. This is the latest iteration of the code splitting initiative. The main goal of code splitting is to reduce the initial download size of a Spartacus enabled app by leveraging lazy loading of feature libraries.
 
-# Backwards compatibility for existing apps by default
+# Existing Apps upgrading to 4.2: keep backwards compatibility
 
-By default, if you upgrade to 4.2 from 4.x, the order library will not be used. Instead, your app will continue to use the old/current order related code. The non-lib order code is still present in Spartacus and will only be removed in 5.0. This allows Spartacus 4.2 to be backwards compatible with previous 4.x versions.
+The the new Order Library is not be backwards compatible. Therefore, by default, if you upgrade to 4.2 from 4.x, the order library will NOT be used. Instead, your app will continue to use the old/current order related code to preserve the backwards coompatibility of the upgrade. The non-lib order code is still present in Spartacus. This allows Spartacus 4.2 to be backwards compatible with previous 4.x versions.
 
 # Order lib used in new Spartcus Apps.
 
-In Spartacus 4.2, if you generate a new Spartacuus app with the schematics, the app will use the new Order Library.
+In Spartacus 4.2, if you generate a new Spartacus app with the schematics, the app will use the new Order Library.
+
+# Order lib used in new Spartcus Apps.
```

Comment:
The title is not correct, we are talking about upgrade, not in new app.

Reply:
We're covering the 4.2 release.  Upgrading to 4.2 and starting a new app with 4.2 are 2 valid use cases, no?
Order lib is the default for the new app use case, while the legacy order code is the default for the upgrade.